docker-compose.yml

```diff
@@ -43,6 +29,13 @@ services:
       options:
         max-size: "1M"
         max-file: "10"
+    networks:
+      - opencdms-surface
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.fastapi.rule=Host(`$HOST_FQDN`)"
```

Comment:
define a HOST_FQDN in bash (for opencdms we set HOST_FQDN=api.opencdms.org, because api.opencdms.org is the domain we want the certificate to be issued for)